plugin/al/usdtransaction/AL/usd/transaction/tests/CMakeLists.txt

```diff
@@ -52,13 +52,18 @@ endif()
 if(IS_MACOSX)
     # Create symbolic link to python framework
     file(CREATE_LINK ${MAYA_LOCATION}/Frameworks ${CMAKE_CURRENT_BINARY_DIR}/../Frameworks SYMBOLIC)
+    file(CREATE_LINK ${MAYA_LOCATION}/Frameworks ${AL_INSTALL_PREFIX}/Frameworks SYMBOLIC)
 endif()
 
-mayaUsd_add_test(GTest:${TARGET_NAME}
-    COMMAND $<TARGET_FILE:${TARGET_NAME}>
-    ENV
-        "LD_LIBRARY_PATH=${ADDITIONAL_LD_LIBRARY_PATH}"
-)
+# Disable the test for now here to simplify tests: MAYA-124144 MayaUsd: Maya OSX blessing failing GTest:AL_USDTransactionTests
+if (NOT MAYA_APP_VERSION VERSION_GREATER 2023 OR NOT IS_MACOSX)
+    mayaUsd_add_test(GTest:${TARGET_NAME}
+        COMMAND $<TARGET_FILE:${TARGET_NAME}>
+        ENV
+            "LD_LIBRARY_PATH=${ADDITIONAL_LD_LIBRARY_PATH}"
+            "DYLD_PRINT_LIBRARIES=YES"
```

Comment:
I should with this line be able to know what is not loading properly on the new build machine.
Once this PR is merged, I will re-enable the test in ECG fraqmework.